governance/release-process.md

```diff
@@ -0,0 +1,61 @@
+# Release Management
+
+[Release manager(s)](roles.md) of stable releases are
+responsible for approving and merging backports, tagging stable releases, and
+sending announcements about them.
+
+# Release Process
+This section describes the release processes for tracking, preparing, and
+creating new Paraglider releases. This includes information around the release cycles
+and guides for developers responsible for preparing upcoming stable releases.
+
+## Active Development
+
+Active development is happening on the `main` branch and the target is to
+release a new version approximately every 6 months.
+
+## Release Tracking
+
+Feature work for upcoming releases is tracked through
+[GitHub Issues](https://github.com/paraglider-project/paraglider/issues).
+
+## Steps to Create a New Release (TODO: Review this more closely)
+1. Bump the version number ([Semantic Versioning 2.0](https://semver.org/))
+  for the following files:
+  * [version.go](../internal/version/version.go)
+2. Create a new pull request for the version number changes
+3. Once the build artifacts complete and all checks pass, download the build
+  artifacts, which can be found via the Actions tab on github
+4. Copy the binary into a VM and manually test user functionality (use the CLI, client, etc.). TODO: Add to CI/CD
```

Comment:
As part of step 2, we should have the CICD in place. at least for VMs. Maybe we strengthen that, then step 4 would be redundant.